### DISCUSSION TITLE
Issue template

### DISCUSSION THREAD

**Discussion by ElieTaillard** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2691] [Ideas]
I am experimenting with issues templates. 
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository
I think it could be a good improvement for the repository. 
![image](https://user-images.githubusercontent.com/54487782/168250418-dec3fe56-390e-49c8-81a0-3095e8da2c21.png)

The functionality that allows to automatically label the issues would save us a lot of time on the issue management.
The issues would be directly sorted.

What do you think about it ?

You can test it in my test repo : https://github.com/Xaalek/test/issues/new/choose

**Comment by Keboo** [comment]
I think this is a great idea. I have been wanting to add in more automation so I love this.

**Comment by ElieTaillard** [comment]
I was thinking about making 3 issue forms:
- Bug report
- Feature request
- Question/Help request

What should be the default labels for each template?

**Reply by Keboo** [reply]
I think probably the following:
- Bug report: bug, evaluation required
- Feature request: enhancement, evaluation required
- Question/Help request: question (it might be worth putting a note here to suggest using discussions for these?)

**Reply by ElieTaillard** [reply]
I submited a pull request
